api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -496,6 +496,27 @@ public void shouldAddTheNecessaryPrivilegesAndAssignThemToSpecificRoles() throws
 		assertTrue(roleHasPrivilege(AUTHENTICATED_ROLE, ADD_VISITS));
 	}
 	
+	@Test
+	public void shouldHaveUpdatedOrderTableInDatabase() throws Exception {
+		String FULFILLER_STATUS = "RECEIVED";
+		String FULFILLER_COMMENT = "Comment";
+		
+		upgradeTestUtil.executeDataset(STANDARD_TEST_1_9_7_DATASET);
+		upgradeTestUtil.executeDataset(UPGRADE_TEST_1_9_7_TO_1_10_DATASET);
+		createOrderEntryUpgradeFileWithTestData("mg=111\ntab(s)=112\n1/day\\ x\\ 7\\ days/week=113\n2/day\\ x\\ 7\\ days/week=114");
+		
+		upgradeTestUtil.upgrade();
+		Connection connection = upgradeTestUtil.getConnection();
+		final String insertOrderQuery = "UPDATE orders SET fulfiller_status='"+ FULFILLER_STATUS + "' , fulfiller_comment = '" +FULFILLER_COMMENT + "' where order_id = 6 ";
+		DatabaseUtil.executeSQL(connection, insertOrderQuery, false);
+		connection.commit();
+		List<Map<String, String>> orders = upgradeTestUtil.select("orders", "order_id = 6 or order_id = 7 and fulfiller_comment != ''",
+			"fulfiller_status", "fulfiller_comment");
+		
+		assertTrue(orders.size() > 0);
+		
+	}
+	
```

Comment:
You don't need this here... this class was specifically for updating the 1.9.7 DB upgrade and some specific updates that happened there.  

If anything, you'd create a Database2_2_0UpgradeIT test... I think this would be a *really* good idea to have one, but, generally, I don't think we've been writing tests for all DB upgrades.  If not, I think that could be handled in a separate ticket which we could create (ie "Create Database 2.2.0 upgrade IT test").  Thoughts @dkayiwa  @wluyima ?